src/st/chi/en_venus_weed.c

```diff
@@ -278,8 +279,11 @@ static u8 AnimFrames_ThornweedDisguise[] = {
 
 // D_80181904
 static u8 AnimFrames_ThornweedQuickWiggle[] = {
-    0x02, 0x2A, 0x02, 0x2B, 0x02, 0x2C, 0x02, 0x2D, 0x00, 0x00, 0x00, 0x00, 0x10, 0x0C, 0x0E, 0x0E,
-    0x0C, 0x12, 0x0E, 0x13, 0x02, 0x14, 0x04, 0x15, 0x20, 0x16, 0xFF, 0x00
+    0x02, 0x2A, 0x02, 0x2B, 0x02, 0x2C, 0x02, 0x2D, 0x00, 0x00, 0x00, 0x00
+};
+
+static u8 AnimFrames_Unused[] = {
+    0x10, 0x0C, 0x0E, 0x0E, 0x0C, 0x12, 0x0E, 0x13, 0x02, 0x14, 0x04, 0x15, 0x20, 0x16, 0xFF, 0x00
 };
```

Comment:
Discovered unused animation here which is stripped on PSP, split this out of the previous `AnimFrames_ThornweedQuickWiggle` array